apl-db-updater/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>apollo-blockchain</artifactId>
+    <groupId>com.apollocurrency</groupId>
+    <version>1.46.2</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>apl-db-updater</artifactId>
+
+  <dependencies>
+
+
+    <!-- Flyway   -->
+    <dependency>
+      <groupId>org.flywaydb</groupId>
+      <artifactId>flyway-core</artifactId>
+      <version>7.0.2</version>
```

Comment:
Can we move flyway version into **Apollo-box-ext** dependency and use it from there, like org.slf4j, org.projectlombok below ?